lexer/impl/lexer.cpp

```diff
@@ -141,6 +142,15 @@ Token Lexer::Next() {
 
     m_pos += lastTerminalLength;
 
+    // I'll use the fact that // can be considered an operator
+    // If we got it, then we'll just skip to the next newline and start over
+    // as if nothing happened.
+    if (tok.type == TokenType::OneLineComment) {
+        for (; m_pos < m_buf.size() && m_buf[m_pos] != '\n'; m_pos++) {}
```

Comment:
```suggestion
        while (m_pos < m_buf.size() && m_buf[m_pos] != '\n') m_pos++;
```
feels a bit more readable